README.md

```diff
@@ -46,6 +46,11 @@ Ensure you are running Linux (kernel 3.8 or later) and have Docker
     sudo curl -L git.io/weave -o /usr/local/bin/weave
     sudo chmod a+x /usr/local/bin/weave
 
+For usage on OSX (with Docker Machine) you first need to make sure
+that a VM is running and configured, as shown in [the Docker Machine
+documentation.](https://docs.docker.com/installation/mac/) Then you
+can launch weave directly from the osx host.
```

Comment:
Hmm. Presumably we need people to follow the "From your shell" rather than "From the Docker Quickstart Terminal" instructions.

Also, missing full stop. and s/osx/OSX.